src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  let discount = 0;
```

Comment:
Prefer using `const` over `let` for the `discount` variable since it is not being reassigned in different scopes, but rather its value is set once based on the condition and used directly.